tests/test_qltk_songmodel.py

```diff
@@ -126,6 +128,20 @@ def test_shuffle(self):
             self.assertEqual(self.pl.current, None)
             self.pl.order.reset(self.pl)
 
+    def test_shuffle_previous_after_reorder(self):
+        rand = Random(7)
+        self.pl.order = OrderShuffle()
+        history = [self.pl.current for _ in range(10) if self.pl.next() or True]
```

Comment:
Why the `or True` - is the `if` clause functioning as a `zip()` here?

Reply:
(I followed the test above for consistency)

This construct is just to make a one-liner out of a for-loop + `history.append(self.pl.current)` and since `self.pl.next()` returns None it needs `or True`.